code/game/objects/structures/ladders.dm

```diff
@@ -10,7 +10,7 @@
 	var/obj/structure/ladder/up     //the ladder above this one
 	var/crafted = FALSE
 	/// Optional travel time for ladder in deciseconds
-	var/travel_time = 0
+	var/travel_time = 100 //SKYRAT EDIT
```

Comment:
Climbing up a Z-Level should not take 10 seconds under any circumstance.
```suggestion
	var/travel_time = 20 //SKYRAT EDIT
```
Two seconds is much more bearable and it's about the same time it takes to climb a ladder on baycode.

Reply:
I'm willing to lower the total amount of time for sure. But I think if I put it took low, people won't feel the need to upgrade anymore. How about six seconds instead, if you don't have it? That is already a 40% reduction on it. I could maybe agree with two seconds, if it wasn't for the fact it would really make the whole skill-chip part of it pointless, which isn't what I want with my plans to encourage people to get more used to using the skill station, along with adding something to upgrade so that people can visit science.

Reply:
If you don't want to make the skillchip redundant, then make it so having it installed makes you climb it in 1 or 2 deciseconds. Anything more than 3 seconds of climbing time just becomes tedious and detrimental to gameplay for no reason other than being tedious.

Reply:
Hmmm. I think you make a decent point, so I'll go ahead and lower the time required as per your suggestions.